tests/lib/c_lib/thrd/testcase.yaml

```diff
@@ -0,0 +1,40 @@
+common:
+  tags:
+    - clib c11 threads
+  filter: not CONFIG_NATIVE_APPLICATION
+  integration_platforms:
+    - qemu_x86
+tests:
+  libraries.libc.c11_threads.minimal:
+    tags: minimal_libc
+    filter: CONFIG_MINIMAL_LIBC_SUPPORTED
+    platform_exclude:
+      - native_posix
```

Comment:
The idea was to put this in the common part of the tests definition

Reply:
Fixed!